<ui version="4.0" >
 <class>EditDesktop</class>
 <widget class="QDialog" name="EditDesktop" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAbout" >
      <attribute name="title" >
       <string>About</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lblName" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="buddy" >
          <cstring>txtname</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtname" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblComment" >
         <property name="text" >
          <string>Comment:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblAuthors" >
         <property name="text" >
          <string>Authors:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblIcon" >
         <property name="text" >
          <string>Icon:</string>
         </property>
         <property name="buddy" >
          <cstring>txticon</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="txticon" />
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="btnSearch" >
         <property name="text" >
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QPlainTextEdit" name="txtcomment" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QPlainTextEdit" name="txtauthors" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="lblStatus" >
       <property name="text" >
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtname</tabstop>
  <tabstop>txtcomment</tabstop>
  <tabstop>txtauthors</tabstop>
  <tabstop>txticon</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
